cabal.project

```diff
@@ -2,12 +2,10 @@ packages: .
 
 -- these upper bounds are required in order to not break payload validation in chainweb
 constraints: base16-bytestring <1
-constraints: prettyprinter <1.6.1
```

Comment:
AFAIK this change breaks mainnet replay. We are currently stuck with an old version of pretty printer.

Reply:
Ok, I'll see if I can avoid this.